packages/cli/test/shared/utils.ts

```diff
@@ -0,0 +1,111 @@
+import { randomBytes } from 'crypto';
+import express = require('express');
+import * as superagent from 'superagent';
+import * as request from 'supertest';
+import { URL } from 'url';
+import bodyParser = require('body-parser');
+
+import config = require('../../config');
+import { AUTHLESS_ENDPOINTS, REST_PATH_SEGMENT } from './constants';
+import { addRoutes as authMiddleware } from '../../src/UserManagement/routes';
+import { Db } from '../../src';
+import { User } from '../../src/databases/entities/User';
+import { meNamespace as meEndpoints } from '../../src/UserManagement/routes/me';
+import { usersNamespace as usersEndpoints } from '../../src/UserManagement/routes/users';
+import { getConnection } from 'typeorm';
+import { issueJWT } from '../../src/UserManagement/auth/jwt';
+
+export const isTestRun = process.argv[1].split('/').includes('jest');
+
+export const MIN_PASSWORD_LENGTH = 8;
+
+export const MAX_PASSWORD_LENGTH = 64;
```

Comment:
we should get them from constants in the app if possible

Reply:
Adding these constants as static properties in `App` means importing that enormous class only for two integers needed during the test run. Are we sure we want to do this? What about in the `User` entity?